chxtools/Mask_Maker.py

```diff
@@ -0,0 +1,131 @@
+from chx_libs import *
```

Comment:
Make these relative imports (`from .chx_libs import *`) and, better yet, import only the parts you need (`from chx_libs import some_func, another_func`) so it is clear to the reader where all the functions below came from.